access/types/forms.py

```diff
@@ -450,7 +454,15 @@ def grade_field(self, i, configuration):
                 else:
                     ok, hints, method = self.grade_checkbox(
                         self.row_options(configuration, row), value, hints)
-                if ok:
+                if self.exercise.get('feedback'):
+                    points += row.get("points", 0)
+                    max_points += row.get("points", 0)
+                elif "partial_points" in self.row_options(configuration, row):
```

Comment:
`partial_points` may be False in the config.yaml so it is not enough to check whether the key exists. You should use the `get` method of dictionaries.